tests/scripts/shepherd/update-gomod-shepherd.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+if [ "$1" == "-h" ] || [ -z "$1" ]; then
+  echo "Usage: `basename $0` <path-to-upstream-shepherd> <shepherd-release-branch> <path-to-rancher>" 
+  echo "WARN: this will update your local go.mod and go.sum files in your rancher repo."
+  printf "Requirements:\n* local clone of upstream rancher/shepherd\n* clone of rancher fork with local changes\n"
+  echo "example: `basename $0` ~/upstream-shepherd release/v2.8 ~/rancher-fork"
+  exit 0
+fi
+
```

Comment:
Might be redundant, but can there be error checking implemented to ensure that that the paths the user specify actually exist? The script reads that it will just error out otherwise, but wouldn't hurt to add.